less/index.less

```diff
@@ -52,8 +52,126 @@ table {
 * {
     box-sizing: border-box;
 }
+// Variables
```

Comment:
Great use of variables, and also very nice with the comment to identify that these are variables. Comments are very important, and will be even more so when you start with JavaScript.